src/GlobalStyles.js

```diff
@@ -20,7 +20,7 @@ export default styled.createGlobalStyle`
   }
 
   body {
-    margin: 0 20px;
+    /* margin: 0 20px; */
```

Comment:
If you comment it out, please remove the code instead, as you don't need them here anymore, do you?

Reply:
Removed